editor/import/resource_importer_texture_atlas.cpp

```diff
@@ -276,9 +277,17 @@ Error ResourceImporterTextureAtlas::import_group_file(const String &p_group_file
 		idx++;
 	}
 
+	//get atlas size limit
+	Ref<ConfigFile> group_file_config;
+	group_file_config.instantiate();
+	Error err = group_file_config->load(p_group_file + ".import");
+	ERR_FAIL_COND_V(err != OK, ERR_FILE_CORRUPT);
+	ERR_FAIL_COND_V(!group_file_config->has_section_key("params", "process/size_limit"), ERR_FILE_CORRUPT);
+	const int atlas_size_limit = group_file_config->get_value("params", "process/size_limit");
+
 	//pack the charts
 	int atlas_width, atlas_height;
-	EditorAtlasPacker::chart_pack(charts, atlas_width, atlas_height);
+	EditorAtlasPacker::chart_pack(charts, atlas_width, atlas_height, atlas_size_limit);
```

Comment:
This breaks if `size_limit` is `0`, which is the default, which means no size limit

Reply:
That's most likely a bug due to the implementation not expecting it, what was the resulting size?

Reply:
Please try with that MRP and see what the results are and how safe it becomes

Reply:
I got it know. Thanks for the explanation.